api-audit/docker/api-audit-properties-builder.sh

```diff
@@ -5,12 +5,8 @@ if [ "$SKIP_PROPERTIES_BUILDER" = true ]; then
   exit 0
 fi
 
-# if we are linked, use that info
-if [ "$MONGO_PORT" != "" ]; then
-  # Sample: MONGO_PORT=tcp://172.17.0.20:27017
-  export SPRING_DATA_MONGODB_HOST=`echo $MONGO_PORT|sed 's;.*://\([^:]*\):\(.*\);\1;'`
-  export SPRING_DATA_MONGODB_PORT=`echo $MONGO_PORT|sed 's;.*://\([^:]*\):\(.*\);\2;'`
-fi
+SPRING_DATA_MONGODB_HOST=${MONGODB_HOST:-db}
```

Comment:
This change is correct and desired but I believe it should be done lower down,

Reply:
Hi,
At line 8 it's the db host, at line 17 it's db name.